app/main.py

```diff
@@ -1,3 +1,38 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+    def __add__(self, other: float | int) -> int | float:
```

Comment:
The return type for `__add__` should be `Distance` instead of `int | float` since the method returns a `Distance` instance.